{"id": "121687", "title": "Bug 121687: Exception thrown when exporting data from table inside a grid cell", "description": "", "OB": "When attempting to export data from a table inside a grid cell, an exception is thrown.", "EB": "When exporting data from two tables, the expected result is that the data is successfully exported. However, the actual result is that an exception occurs in the ReportEngineService.getMetaData() function.", "SR": "To reproduce this issue, follow these steps: 1. Insert a grid. 2. Export data from the table inside a grid cell."}